angular-workspace/projects/ni/nimble-angular/src/thirdparty/testing/browser_util.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * [Nimble]
+ * Copied from https://github.com/angular/angular/blob/6070c9ddcff88d4ad4bcf73a2dd1874920661d93/packages/platform-browser/testing/src/browser_util.ts
```

Comment:
No, I think I mistakenly pulled both this file and matchers.ts from a recent `main` head revision instead of the 15.2 branch. I'll update them accordingly. Since these files both only affect the copied tests, it should not invalidate any previous testing of the actual upgrade (other than those automated tests).